cucumber/policy/features/step_definitions/policy_steps.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-Given(/^a policy:$/) do |policy|
+Given(/^(?:I load )?a policy:$/) do |policy|
```

Comment:
I would prefer to make the "I load" mandatory here and just update all the places this is used, but I imagine that requires changing A LOT of files so if that's a pain forget about it for now. Definitely better things to do.